osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSubScreen.cs

```diff
@@ -81,134 +82,138 @@ protected override void LoadComplete()
         {
             RelativeSizeAxes = Axes.Both,
             Padding = new MarginPadding { Horizontal = 5, Vertical = 10 },
-            Child = new GridContainer
+            Child = new OsuContextMenuContainer
```

Comment:
Heavily unsure about putting these at a screen level. They are usually nested inside of scroll views so they correctly dismiss context menus when the item displaying them scrolls into a masked away state.

Reply:
I've read through `ContextMenuContainer` and I'm not sure context menus dismiss at all when items are masked away? It only happens when they are no longer present (perhaps an oversight?)

In addition, I've tested using `ParticipantsList` with three different ways of placing the context menu container just to figure out what seems more correct:

 - Placed outside `ParticipantsList`'s scroll content and filling entire screen (current code in PR):
   
   https://user-images.githubusercontent.com/22781491/179121234-cc63438d-afb8-401a-aed1-531852fafa78.mp4

 - Placed outside `ParticipantsList`'s scroll content and filling to the list size (master):
 
   https://user-images.githubusercontent.com/22781491/179121280-5d8b7ca7-10be-41d1-bf21-0b0d3b107061.mp4

 - Placed inside `ParticipantsList`'s scroll content:
   
   https://user-images.githubusercontent.com/22781491/179121342-53afabf1-c61a-4fb1-a4ed-0cf4e84d03e2.mp4

Posting this since it looks more correct to place outside the list and filling entire playlists/multiplayer screen, so that the context menu still displays at the cursor's position when right-clicking the last displayable item.